posts/2015-07-25-new-javascript-generators.md

```diff
@@ -0,0 +1,120 @@
+---
+title: New JavaScript Generators
+author: Freezeboy
+date: 2015-07-25 00:00
+toc: true
+---
+
+# Summary
+
+With the upcoming Servant version, it will be way easier to build a javascript
+connector to your webservice. The chosen API is *as usual with Servant*
+putting the magic away from your eyes to let you concentrate on what you are
+working on in most situations.
+
+Extensibility is still available, again *as usual with Servant*, each part of
```

Comment:
Don't say this till the end. Right now it's not clear what the axis of extensibility is.